src/wallet/wallet_rpc_server.cpp

```diff
@@ -581,6 +584,24 @@ namespace tools
       throw wallet_rpc_error{error_code::NOT_OPEN, "No wallet file"};
   }
   //------------------------------------------------------------------------------------------------------------------------------
+  void wallet_rpc_server::close_wallet(bool save_current)
+  {
+    if (m_wallet)
+    {
+      LOG_PRINT_L0(tools::wallet_rpc_server::tr("Closing wallet..."));
+      stop_long_poll_thread();
+      if (save_current)
+      {
+        LOG_PRINT_L0(tools::wallet_rpc_server::tr("Saving wallet..."));
+        m_wallet->store();
+        LOG_PRINT_L0(tools::wallet_rpc_server::tr("Successfully saved"));
+      }
+      m_wallet->deinit();
+      m_wallet.reset();
+      LOG_PRINT_L0(tools::wallet_rpc_server::tr("Successfully closed"));
```

Comment:
These `L0`s look like leftover noisy debug; they probably belong as `MINFO`s at most.

Reply:
Sure, can do, I just kept them from the original implementation around line 3514. I'll drop them